src/helpers/dimensions.ts

```diff
@@ -11,6 +11,13 @@ export function findBucketByLocalIdentifier(buckets: VisualizationObject.IBucket
     return (buckets || []).find(bucket => bucket.localIdentifier === bucketName);
 }
 
+export function getBucketItems(
```

Comment:
I would like to remove this helper and 'findBucketByLocalIdentifier' to some other file and keep here only functions around dimensions. After merge of app-components to react-components, there is file bucketHelper.ts, BUT it works with different interfaces (with UI buckets).
So probably something like 'mdObjBucketHelper' would be good place for these two helpers